Makefile

```diff
@@ -52,8 +51,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) crd:generateEmbeddedObjectMeta=true paths="./..." output:crd:artifacts:config=deploy/crds
-	$(CONTROLLER_GEN) crd:generateEmbeddedObjectMeta=true paths="./..." output:crd:artifacts:config=chart/templates/crds/
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
question: when i do ```make manifests``` the above command generate the CRD for both config/crd/bases and helm chart crds. How is it going to be with these changes ?

Reply:
It will generate the entire config directory. It won't touch the chart at all so this will still be a manual step. But we need a solution to that. I don't think there is something like that out there.

Reply:
This is Kubebuilder's default

Reply:
Yes, so to solve this i had added
```	$(CONTROLLER_GEN) crd:generateEmbeddedObjectMeta=true paths="./..." output:crd:artifacts:config=chart/templates/crds/```
So this way ```make manifests``` updates both the places. Lets keep it this way :)